doc/releasenotes/17_0_0_summary.md

```diff
@@ -14,6 +14,8 @@
 
 ### <a id="breaking-changes"/>Breaking Changes
 
+ - When using TLS with `vtgr`, we now default to TLS 1.2 if no other explicit version is configured. Configuration flags are provided to explicitly configure the minimum TLS version to be used. 
+
```

Comment:
I think this should be in its own subheading (`####`) under `Breaking Changes`. It's easier for readers to read when they're skimming through the summary at the top of the file.

Reply:
@frouioui What should the title be then? Any example I could look at?

Reply:
We could use something like `#### <a id="tls-version"/>TLS Version Change in Vtgr`